### ISSUE TITLE
[docs] Add examples

### ISSUE THREAD

**Issue by svx** [testthedocs/vale-styles#11]
**Is your feature request related to a problem? Please describe.**
For a person **new** to vale and its configuration, it can be a challenging task to configure and setup all the checks.

Besides the docs on https://errata-ai.github.io/vale/styles/, lots of the checks *do not* give any "easy" examples.
We should add text examples as a comment to every check.
In that way, users who are not wizards with "regex" can actually understand what the check is doing and how it looks like.

**Describe the solution you'd like**
Add a comment to every check and/or the docs explaining better what the check is doing with an example.

**Comment by ChrisChinchilla** [comment]
Yeah, I've wanted to do this for a while, and am about to create a style guide guide(!) for a client, so might use this as a good motivation to do this.

**Comment by svx** [comment]
I started a bit already, not finished yet.
The way I am doing it now, is to create a "simple" README as overview and also add examples as comments to the `yaml` files itself: https://github.com/testthedocs/vale-styles/tree/master/ttd-light

Not sure, yet if that is a good approach but at least it feels better so far :)
